src/marker.js

```diff
@@ -12,12 +12,11 @@ export default class Popup extends React.Component {
     anchor: OverlayPropTypes.anchor,
     offset: OverlayPropTypes.offset,
     children: PropTypes.node,
```

Comment:
It'd be great to get all the missing callback `propTypes` in? Is there any reason why they were missing in the first place?